tests/sequencing_center/test_seq_center_models.py

```diff
@@ -0,0 +1,127 @@
+from datetime import datetime
+import uuid
+import random
+
+from dataservice.extensions import db
+from dataservice.api.sequencing_experiment.models import SequencingExperiment
+from dataservice.api.sequencing_center.models import SequencingCenter
+from tests.utils import FlaskTestCase
+
+from sqlalchemy.exc import IntegrityError
+
+MAX_SIZE_MB = 5000
+MIN_SIZE_MB = 1000
+MB_TO_BYTES = 1000000000
+
+
+class ModelTest(FlaskTestCase):
+    """
+    Test database model
+    """
+
+    def create_seqexp_seqcen(self):
+        """
+        create a sequencing experiment and sequencial center save to db
+        returns sequencing_center kf_id
+        """
+        se_id = "Test_SequencingExperiment_0"
+        seq_experiment_data = self._make_seq_exp(external_id=se_id)
+        se = SequencingExperiment(
+            **seq_experiment_data)
+        db.session.add(se)
+        db.session.commit()
+        sc = SequencingCenter(name="Baylor", sequencing_experiment_id=se.kf_id)
+        db.session.add(sc)
+        db.session.commit()
+        ids = {'sequencing_center_id': sc.kf_id}
+        return ids
+
+    def test_create_and_find_sequencing_center(self):
+        """
+        Test creation of sequencing_center
+        """
+        dt = datetime.now()
+        ids = self.create_seqexp_seqcen()
+
+        self.assertEqual(SequencingCenter.query.count(), 1)
+        sc = SequencingCenter.query.one()
+
+        self.assertEqual(sc.name, "Baylor")
+        self.assertGreater(sc.created_at, dt)
+        self.assertGreater(sc.modified_at, dt)
+        self.assertIs(type(uuid.UUID(sc.uuid)), uuid.UUID)
+
+    def test_update_sequencing_center(self):
+        """
+        Test Updating sequencing_center
+        """
+        ids = self.create_seqexp_seqcen()
+        # get sequencing_center
+        sc = SequencingCenter.query.one_or_none()
+
+        sc.name = "Baylor"
+        db.session.commit()
+
+        # get updated sequencing_center
+        e = SequencingCenter.query.one_or_none()
+        self.assertEqual(e.name, "Baylor")
+
+    def test_delete_sequencing_center(self):
```

Comment:
This test should check whether relations were properly managed (if cascade delete enabled for orphans on a related entity, then those related entities should have been deleted. if cascade delete was not enabled for orphans - which I don't think it should be - then none of the related entities should be deleted when a seq center is deleted.)